<launch>


 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js1" />
    <param name="deadzone" value="0.05" />
  </node>

 <!-- Jogging Axes from the 3d mouse-->
  <param name="axis_linear" value="0" type="int"/>
  <param name="axis_angular" value="1" type="int"/>
  

  <!-- Mode select from the buttons -->
  <param name="mode_select" value="1" type="int"/> <!-- Trigger -->

  <param name="scale_angular" value="10" type="double"/>
  <param name="scale_linear" value="10" type="double"/>
  
  <!-- Select the moveit configuration to use -->
  <param name="movit_config" value="irb1600id_moviet_config" type="string"/>

  <!--- And launch the spacenav control node -->
  <node pkg="spacenav_control" type="spacenav_control" name="teleop"/>

</launch>